src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -195,4 +195,9 @@
   <data name="RazorPage_InvalidTagHelperIndexerAssignment" xml:space="preserve">
     <value>Unable to perform '{0}' assignment. Tag helper property '{1}.{2}' must not be null.</value>
   </data>
+  <data name="CouldNotResolveApplicationRelativeUrl_TagHelper" xml:space="preserve">
+    <value>Could not resolve application relative URL '{0}' by calling '{1}.{2}'. To disable application relative URL resolution you can add the following directive to your Razor page:
+
+@removeTagHelper "{3}, {4}".</value>
```

Comment:
Given the current focus of the message, the `IUrlHelper.Content()` call is an implementation detail of the `UrlResolutionTagHelper`. Should remove `by calling '{1}.{2}'` if the message otherwise remains unchanged.

The period in the code sample is incorrect and should also be removed.

But resolution was probably fine and removing the `UrlResolutionTagHelper` is probably incorrect for this case. It's more likely the user has overridden the `IUrlHelper` in DI, replacing it with something that does more than change the prefix.

All up
```
Unexpected return value from '{1}.{2}' for URL '{0}'. If you have overridden the '{1}' service, change '{2}' to replace only the '~' prefix. Otherwise, add the following directive to your Razor page to disable application-relative URL resolution:

@removeTagHelper "{3}, {4}"
```